### ISSUE TITLE
Address lookup will fail for upgrader

### ISSUE THREAD

**Issue by clexmond** [rocket-pool/rocketpool#13]
Really appreciate the work done on eternal storage / upgradeable contracts. In going through code review I'm fairly certain the upgrader will always throw because the old contract address will never be found. The lookup currently uses the `rocketStorage` key when I believe it should be using `contract.name`:

https://github.com/rocket-pool/rocketpool/blob/34e222d992ea6b014620978a2f7d6ed2c495dacc/contracts/RocketUpgrade.sol#L37

**Comment by darcius** [comment]
Hey @clexmond! Indeed you're right mate, that `rocketContract` key was actually the old key we used to store contract addresses with, before we changed it too `contract.name` - looks like it missed getting updated, so I've fixed that now. Appreciate the heads up.

There's still a TODO in that contract that involves getting the unit tests finished for it, might have to bump that up a few notches on the work ladder :)
